components/keyrings/common/operations/operations.h

```diff
@@ -631,12 +631,6 @@ class Keyring_operations {
   */
   bool generate(const meta::Metadata &metadata, const data::Type type,
                 size_t length) {
-<<<<<<< HEAD
-    Data_extension generated_data(keyring_common::data::Data{"", type});
```

Comment:
we were explicitly creating the data object and then passing it to generated_data. I do not why we did that. But now upstream is also doing the same. That's why dropped our changes. It look similar to me

`data::Data g_data(type);
  Data_extension generated_data(g_data);`

Reply:
@rahulmalik87 Check with Zsolt and Yura about this one.
Do we have a jenkins job for this merge?

Reply:
We changed it, because the original code was incorrect - and looks like upstream realized this too.

`Data_extension` is a template parameter with a default type `Data`. The original code only compiled in this case, a custom `Data_extension` resulted in a compilation error. This fix (both ours and upstream) fixes the issue.

Reply:
> @rahulmalik87 Check with Zsolt and Yura about this one.
> Do we have a jenkins job for this merge?

No Marce. The AWS spot instances are getting killed. I think because of Thanksgiving. Started again. I have old [jenkin.](https://pxb.cd.percona.com/view/PXB%208.0/job/percona-xtrabackup-8.0-multijob/212/) which doesn't contain the latest fix.  So one unknown issue pxb-1679.sh. which I saw in the feature branch also and is based on 8.0.30. 
So not new. But I can analyze pxb-1679.sh after Keep_dir -k option to see what was wrong.